readme.md

```diff
@@ -1,28 +1,28 @@
-Nette Sandbox
+Baraja Nette Sandbox
 =============
-
-This is a simple pre-packaged and pre-configured application using the [Nette](https://nette.org)
-that you can use as the starting point for your new applications.
-
 [Nette](https://nette.org) is a popular tool for PHP web development.
 It is designed to be the most usable and friendliest as possible. It focuses
 on security and performance and is definitely one of the safest PHP frameworks.
 
-If you like Nette, **[please make a donation now](https://nette.org/donate)**. Thank you!
+This is an upgraded version of the the basic Nette Sandbox that you can use as the starting point for your new applications.
+The PROs of this Sandbox is that is featuring the https://github.com/baraja-core/package-manager a great tool that allows you to search all the package dependencies automatically and register them to your project.
+It is also fully configured and also prepared to support the use of doctrine(https://github.com/baraja-core/doctrine) which is a simple and easy to use, maximal performance database layer.
 
 
-Installation
-------------
 
-The best way to install Web Project is using Composer. If you don't have Composer yet,
-download it following [the instructions](https://doc.nette.org/composer). Then use command:
 
-	composer create-project nette/sandbox path/to/install
-	cd path/to/install
 
+Installation
+------------
+To install the sandbox you should use Composer. To do so, find your web root directory (e.g. /var/www or C:\InetPub) in your command line and execute the following command:
 
-Make directories `temp/` and `log/` writable.
+composer create-project baraja/sandbox <your directory>
```

Comment:
Please change `<your directory>` to `<your-project-name>`

Reply:
Completed